src/platform/forms/save-in-progress/RoutedSavablePage.jsx

```diff
@@ -88,6 +89,7 @@ function mapStateToProps(state, ownProps) {
   return {
     form: state.form,
     user: state.user,
+    isLoggedIn: isLoggedIn(state),
```

Comment:
Not at the moment, however it could probably be put in [here](https://github.com/department-of-veterans-affairs/vets-website/blob/4e552c397a7effc35e87c4fc4505f62880b50266/src/platform/forms/save-in-progress/RoutedSavablePage.jsx#L36)